config/initializers/decidim.rb

```diff
@@ -25,11 +25,13 @@
   config.available_locales = %i(ca es en)
 
   # Geocoder configuration
-  # geocoder_config = Rails.application.secrets.geocoder
-  # if geocoder_config&.dig(:here_api_key).present?
-  #   config.geocoder = {
-  #   }
-  # end
+  geocoder_config = Rails.application.secrets.geocoder
+  if geocoder_config[:here_app_id].present? && geocoder_config[:here_app_code].present?
+    config.geocoder = {
+      here_app_id: geocoder_config[:here_app_id],
+      here_app_code: geocoder_config[:here_app_code]
+    }
+  end
```

Comment:
Geocoder was intentionally commented as we are not using it.
It is good that you upgrade the configuration but please comment it again.